src/sentry/api/urls.py

```diff
@@ -497,6 +498,11 @@
         ProjectEnvironmentsEndpoint.as_view(),
         name='sentry-api-0-project-environments'
     ),
+    url(
+        r'^projects/(?P<organization_slug>[^\/]+)/(?P<project_slug>[^\/]+)/environments/(?P<environment>[^/]+)/$',
+        ProjectEnvironmentDetailsEndpoint.as_view(),
+        name='sentry-api-0-project-environment-details'
+    ),
```

Comment:
Naming here for the slug parameter as well as the correct pluralization of the class name and route name is all over the place so I just chose one

Reply:
This is a new endpoint that'll fuck up if we have environment name with a `/` which iirc wasn't unusual.

We should prioritize making stricter allowance on names soon.

Reply:
I agree, I did this through gritted teeth for consistency's sake